StortSpelProjekt/Engine/src/ECS/Components/Component.cpp

```diff
@@ -20,8 +20,19 @@ void Component::RenderUpdate(double dt)
 {
 }
 
-void Component::InitScene()
+void Component::OnInitScene()
 {
+	Log::PrintSeverity(Log::Severity::WARNING, "COMPONENT OnInitScene called! Declare and define your own OnLoadScene in your component class.\n");
+}
+
+void Component::OnLoadScene()
+{
+	Log::PrintSeverity(Log::Severity::WARNING, "COMPONENT OnLoadScene called! Declare and define your own OnLoadScene in your component class.\n");
+}
+
+void Component::OnUnloadScene()
+{
+	Log::PrintSeverity(Log::Severity::WARNING, "COMPONENT OnUnloadScene called! Declare and define your own OnUnloadScene in your component class.\n");
```

Comment:
These prints will be very annoying if some components aren't meant to have their own function definitions. In the end, these might print 1000 times and completely destroy the output window.

Reply:
fixed